locales/translation-de.json

```diff
@@ -260,6 +260,8 @@
   "groups": {
     "contact": "Ansprechpartner",
     "contacts": "Ansprechpartner",
+    "contact_the_organizers": "Kontaktieren",
+    "contactTheOrganizers_label": "Gäste können Ansprechpartner kontaktieren",
```

Comment:
"Gäste" finde ich zu unspezifisch. Im Moment sollte es lauten "Softwerkskammer-Mitglieder", oder?

Reply:
Im finalen Ausbau soll das auch für Gäste möglich sein. Unsere Überlegung war, wir wollten die gleiche Konfiguration weiterverwenden. Und damit auch im jetztigen Zustand gleich klar kommuniziert wird für was das da sein soll damit später keiner Überrascht wir, wenn Mails von Gästen kommen

Reply:
Da wir hier gleichzeitig von zwei verschiedenen Dingen sprechen, lass uns das mal trennen:

1. Den Begriff "Gäste" finde ich zu unspezifisch. Ich glaube in Diskussionen haben Andreas und ich immer von "Mitglied" vs. "unangemeldeter Besucher" gesprochen. Klingt das plausibel für Euch?

2. "gleich klar kommuniziert wird" -> Wird es im Moment eben gerade nicht. Ihr sagt, das Feature ist für etwas gut, aber wenn man dann schaut, findet man heraus, dass das nicht stimmt. Das finde ich ziemlich verwirrend... Wie wäre es mit "Softwerkskammer-Mitglieder (und später auch unangemeldete Besucher) können ..."

Reply:
Vorschlag: "Ermögliche, Ansprechpartner anzuschreiben"
Vorteil: 1. Kein "kontaktieren" mehr, 2. Ausbaufähig

Zu Nicole: "Gast" ist ein nicht registrierte Sitebesucher. - Solange das Feature nicht für jeden implementiert ist, ist es y.a.g.n.i. (und ich habe Bedenken anzumelden; es durchbricht das Sicherheitsprinzip, dass E-Mail Adressen nie sichtbar und nur für Mitglieder zur indirekten Kontaktaufnahme dienen)

Reply:
"es durchbricht das Sicherheitsprinzip": Was genau meinst Du hier? Auch wenn Mitglieder miteinander Kontakt aufnehmen, wird sofort die Mailadresse des Sendenden übermittelt.

Reply:
Hier meine ich, dass es

1. Nur registrierten Mitgliedern möglich ist, zu senden. Und 
1. Sie die Empfängeradresse niemals sehen
1. Absenderadresse als reply-to für den "versteckten" Empfänger sichtbar ist.

Das "Konzept" liegt darin, dass es so kommuniziert und bekannt ist. Weiterhin wäre dies ein Einfallstor für Aktionen, die ein nicht-registrierter Sitebesucher vornehmen kann. Aktuell schützen wir quasi alle Aktionen mit "isRegistered"

Reply:
> Gäste vs "unangemeldeter Besucher"

Wenn das eure bisherige Sprache ist, mach das für mich Sinn.

> "Softwerkskammer-Mitglieder (und später auch unangemeldete Besucher) können ..."

Finde ich gut. Dokumentiert den ist Zustand und gibt genug Informationen was es in der Zukunft bedeuten soll das man das gleiche Flag weiterverwenden kann.

> "Ermögliche, Ansprechpartner anzuschreiben"

Um den Punkt von Nicole aufzugreifen, ich kann ja jetzt auch schon Ansprechpartner anschreiben. Nur nicht alle auf einmal. 

> Weiterhin wäre dies ein Einfallstor für Aktionen, die ein nicht-registrierter Sitebesucher vornehmen kann. Aktuell schützen wir quasi alle Aktionen mit "isRegistered"

Die Idee hinter dem Feature ist schon, das nicht angemeldete Besucher die Ansprechpartner kontaktieren können. Das ist noch nicht implementiert, wollen wir aber noch machen. Die emails der Ansprechpartern bekommt der nicht angemeldete Besucher nie zu sehen. Damit die Ansprechpartner Kontakt aufnehmen kann, muss er eine email Adresse mit angeben, die dann auch wieder nur die Ansprechpartner sehen. Das ganze kann noch ueber ein Captcha abgesichert werden.

Ist das etwas, was wir deine Meinung nach nicht anbieten sollten?

Reply:
> > Gäste vs "unangemeldeter Besucher"
>
> Wenn das eure bisherige Sprache ist, mach das für mich Sinn.

@signed Andreas hat mich ja zu Recht hier korrigiert; wir verwenden sehr wohl den Begriff "Gast". (Siehe auch rechts oben in der SWK.) Passt also dann für mich.

@leider Zum "Sicherheitsprinzip": Die Punkte 2 und 3 wären ja auch weiterhin so. Punkt 1 würde dahingehend aufgebohrt, dass Ansprechpartner _nach vorherigem Opt-In_ auch von Nichtmitgliedern angeschrieben werden könnten. Das passt soweit für mich. Wo genau hast Du hier noch Bedenken?
@leider Was das "Aktionen auslösen können" angeht: Das Ganze muss natürlich mindestens hinter einem Captcha sein. Wäre das für Dich ausreichend? Wenn nicht: Was müsste für Dich noch gegeben sein?

@signed "Damit die Ansprechpartner Kontakt aufnehmen kann, muss er eine email Adresse mit angeben" -> Es genügt aber nicht, das in den Fließtext zu packen. Dafür muss ein eigenes Feld vorgesehen werden. Denn die versendete Mail braucht auch eine Absenderadresse. Dafür nehmen wir aktuell die Mailadresse des Mitglieds.